Session/Conversations/Settings/ThreadSettingsViewModel.swift

```diff
@@ -555,215 +964,103 @@ class ThreadSettingsViewModel: SessionTableViewModel, NavigatableStateHolder, Ob
-                        id: .blockUser,
+                        id: .debugDeleteAttachmentsBeforeNow,
                         leadingAccessory: .icon(
-                            UIImage(named: "table_ic_block")?
-                                .withRenderingMode(.alwaysTemplate)
+                            Lucide.image(icon: .trash2, size: 24)?
+                                .withRenderingMode(.alwaysTemplate),
+                            customTint: .danger
                         ),
-                        title: "deleteAfterGroupPR1BlockThisUser".localized(),
-                            trailingAccessory: .toggle(
-                            threadViewModel.threadIsBlocked == true,
-                            oldValue: (previous?.threadViewModel?.threadIsBlocked == true),
-                            accessibility: Accessibility(
-                                identifier: "Block - Switch"
-                            )
-                        ),
-                        accessibility: Accessibility(
-                            identifier: "\(ThreadSettingsViewModel.self).block",
-                            label: "Block"
+                        title: "[DEBUG] Delete all arrachments before now",    // stringlint:disable
+                        styling: SessionCell.StyleInfo(
+                            tintColor: .danger
                         ),
                         confirmationInfo: ConfirmationModal.Info(
-                            title: (threadViewModel.threadIsBlocked == true ?
-                                "blockUnblock".localized() :
-                                "block".localized()
-                            ),
-                            body: (threadViewModel.threadIsBlocked == true ?
-                                .attributedText(
-                                    "blockUnblockName"
-                                        .put(key: "name", value: threadViewModel.displayName)
-                                        .localizedFormatted(baseFont: ConfirmationModal.explanationFont)
-                                ) :
-                                .attributedText(
-                                    "blockDescription"
-                                        .put(key: "name", value: threadViewModel.displayName)
-                                        .localizedFormatted(baseFont: ConfirmationModal.explanationFont)
-                                )
-                            ),
-                            confirmTitle: (threadViewModel.threadIsBlocked == true ?
-                                "blockUnblock".localized() :
-                                "block".localized()
-                            ),
+                            title: "delete".localized(),
+                            body: .text("Are you sure you want to delete all attachments (and their associated messages) sent before now for all group members?"),   // stringlint:disable
```

Comment:
Is there a localised string for this? (probably shouldn't have `stringlint:disable` here if not because we would want this warning until the string is replaced with a localised one)

Reply:
I don't think there is a localised string for this one, and it is still just for DEBUG? I refactored the previous logic and just moved it here since the other functionalities will not be only for DEBUG after this project.

Reply:
Oh sorry, I misread the diff and thought this one was for the "delete messages before now" action rather than the attachments one 🤦 

I'll mark this as resolved